src/exampleTypes.js

```diff
@@ -272,51 +272,38 @@ export default F.stampKey('type', {
           (type !== 'numberRanges' && type !== 'percentiles') ||
           (type === 'numberRanges' && ranges.length > 0) ||
           (type === 'percentiles' && percents.length > 0),
-        _.concat(context.columns, context.groups)
+        _.concat(context.columns, context.rows)
       ),
     reactors: {
       columns: 'self',
-      groups: 'self',
+      rows: 'self',
       values: 'self',
       drilldown: 'self',
       filters: 'others',
       sort: 'self',
-      flatten: 'self',
```

Comment:
Could I get a comment on why this can be removed?  I'm assuming the same field is removed below for the same reason.

Reply:
After talks it sounds like flatten has been depricated entirely from contexture